packages/fx-core/resource/package.nls.json

```diff
@@ -297,8 +297,7 @@
   "core.createProjectQuestion.projectType.bot.detail": "Conversational or informative chat experiences that can automate repetitive tasks",
   "core.createProjectQuestion.projectType.bot.label": "Bot",
   "core.createProjectQuestion.projectType.bot.title": "App Features Using a Bot",
-  "core.createProjectQuestion.projectType.messageExtension.detail": "Search and take actions from the text input box in Teams and Outlook",
-  "core.createProjectQuestion.projectType.messageExtension.copilotEnabled.detail": "Search or initiate actions from the message composing area of Teams, Outlook and Copilot",
+  "core.createProjectQuestion.projectType.messageExtension.agentEnabled.detail": "Search or initiate actions from the message composing area of Teams, Outlook and Agent",
```

Comment:
Proposed: Search or initiate actions from the message input box in Teams, Outlook, and Agent.

Reply:
Updated. Thanks